src/components/Inngest.ts

```diff
@@ -328,7 +328,7 @@ export class Inngest<
     // dev server.
     let url = this.inngestApiUrl.href;
 
-    if (!isProd()) {
+    if (!isPlatformDeploy()) {
```

Comment:
As discussed elsewhere (cc @goodoldneon), keeping `isProd()` here (or adding a new `useDevServer()` function) would be good.